storey/flow.py

```diff
@@ -78,6 +78,17 @@ def _user_fn_output_to_event(self, event, fn_result):
 
 
 class Choice(Flow):
+    """Redirects each input element into at most one of multiple downstreams.
+
+    :param choice_array: a list of (downstream, condition) tuples, where downstream is a step and condition is a function. The first
+    condition in the list to evaluate as true for an input element causes that element to be redirected to that downstream step.
+    :type choice_array: tuple of (Flow, Function (Event=boolean))
```

Comment:
`Event=boolean` ?
I think the syntax for a function type should be something like `callable[[args..], result]`

Reply:
Was meant to be => like everywhere else.